src/lib/containers/UserCard.tsx

```diff
@@ -62,48 +63,67 @@ export const UserCard: React.FunctionComponent<UserCardProps> = (
     ...rest
   } = props
   const { accessToken } = useSynapseContext()
-  const [userProfile, setUserProfile] = useState(initialProfile)
   const [principalId, setPrincipalId] = useState(ownerId)
   const [isLoading, setIsLoading] = useState(true)
-  const [preSignedURL, setPresignedUrl] = useState<string | undefined>(
-    initialPreSignedURL,
-  )
+  const [fetchedPresignedUrl, setFetchedPresignedUrl] = useState<
+    string | null | undefined
+  >(undefined)
 
-  const { data: profileAndImage } = useGetUserProfileWithProfilePic(
-    principalId!,
-    {
-      enabled: !!principalId,
-    },
-  )
+  const { data: fetchedProfile } = useGetUserProfile(principalId!, {
+    enabled: !!principalId && !initialProfile,
+  })
+
+  const userProfile = initialProfile ?? fetchedProfile
```

Comment:
Only fetch a user profile if we weren't provided an initialProfile object (and have a principal ID)

Then, define `userProfile` as the initial profile, falling back to the fetched profile, if it exists.